src/common/ceph_argparse.cc

```diff
@@ -132,6 +132,12 @@ void argv_to_vec(int argc, const char **argv,
   args.insert(args.end(), argv + 1, argv + argc);
 }
 
+std::vector<const char*> argv_to_vec(int argc, const char** argv)
+{
+  std::vector<const char*> args{argv+1, argv+argc};
+  return args;
```

Comment:
```suggestion
  assert(argc > 1);
  return {argv + 1, argv + argc};
```

Reply:
I really like the return.
The assert is not, I think, a good idea. The code handles argc==1 correctly
(going to verify that shortly)

Reply:
how about `argc > 0` ?